### PR TITLE
BUGFIX: remove last MultiFormSessionID

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

File: code/model/MultiForm.php [hunk 1]
```diff
@@ -176,6 +176,15 @@ public function getController() {
 	}
 
 	/**
+	 * Returns the get_var to the template engine
+	 *
+	 * @return string
+	 */
+	public function get_var() {
```
Comment (tractorcow) [inline code/model/MultiForm.php]: Is this the proper naming convention for public methods? It should be lower camel case (psr-2 and standard ss convention).

**Comment by tractorcow** [comment]
I assume this is fixing https://github.com/silverstripe/silverstripe-multiform/issues/51#issuecomment-160569764? Can you fix the breaking test too please?

File: code/model/MultiForm.php (hunk 1 above)
Comment (spekulatius) [inline code/model/MultiForm.php]: I can adjust it to be consistent with the naming which would be "getGetVar" - pretty weird name. What do you think @tractorcow ?

**Comment by spekulatius** [comment]
Hello @tractorcow ,

I fixed the test and the name of the method. Do you want to merge the two pull requests in and tag it as 1.2.1?

spek

**Pull request merged**